### PR TITLE
Fix issue with Project share

### PR DESCRIPTION
**Pull request by simonebelotti75**

### CODE REVIEW

File: src/routers/AppRouter.js [hunk 1]
```diff
@@ -71,7 +70,7 @@ const AppRouter = () => {
           path="/project/:projectId/preview"
           component={PreviewPage}
         />
-        <PrivateRoute
+        <Route
```
Comment (muhammadqamar) [inline src/routers/AppRouter.js:73]: IT will be good to use the OpenRoute component of public routes

File: src/routers/AppRouter.js [hunk 2]
```diff
@@ -96,14 +95,12 @@ const AppRouter = () => {
           component={PreviewPage}
           previewType="playlist"
         />
-        {/*
-        <PrivateRoute
+        <Route
```
Comment (muhammadqamar) [inline src/routers/AppRouter.js:98]: same here

File: src/routers/OpenRouter.js [hunk 3]
```diff
@@ -1,22 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Route } from 'react-router-dom';
-
-const OpenRouter = ({ component: Component, path, previewType }) => (
```
Comment (muhammadqamar) [inline src/routers/OpenRouter.js:5]: whe we delete this ?

**Review by muhammadqamar — CHANGES_REQUESTED** [review]
kindly revert the changes for openRoute component, This component is for public routes

File: src/routers/AppRouter.js (hunk 1 above)
Comment (simonebelotti75) [reply src/routers/AppRouter.js:73]: No need OpenRoute
I removed that component at all

File: src/routers/OpenRouter.js (hunk 3 above)
Comment (simonebelotti75) [reply src/routers/OpenRouter.js:5]: It is same as Route component, we don't need this

**Review by muhammadqamar — APPROVED** [review]

**Pull request merged**